mesmerize_viz/_store_model.py

```diff
@@ -40,7 +40,7 @@ def __init__(self, subscriber, data=None, data_filter=None, multiplier=None):
         # must have data if ImageGraphic
         if isinstance(self.subscriber, (ImageGraphic, ScatterGraphic)):
             # may prefer to check for hasattr(var, 'shape'), to allow dask/zarr/other numpy-like objects
-            if not isinstance(data, np.ndarray):
+            if not hasattr(data, 'shape'):
```

Comment:
I think checking for `__array__` might be better

Reply:
[rcm lazy array](https://mesmerize-core.readthedocs.io/en/latest/api/lazy_arrays/rcm.html) has no `__array__` so this check fails